Mono.Cecil.Cil/Symbols.cs

```diff
@@ -802,6 +809,45 @@ public ISymbolReader GetSymbolReader (ModuleDefinition module, string fileName)
 			return null;
 		}
 
+		public static SymbolReaderKind GetSymbolReaderKind(string fileName)
+		{
+			try
+			{
```

Comment:
If we are going to do a try/catch all exceptions, would a Try method pattern be better to follow?  i.e. along the lines of Dictionary.TryGetValue.

So the signature would be `public static bool TryGetSymbolReaderKind(string fileName, out SymbolReaderKind  kind)`

Alternatively, maybe we could loosen up the catch?  What failure case are we trying to guard against?  No symbol files?  Not a managed assembly?